core/server/common/src/main/java/alluxio/master/BackupManager.java

```diff
@@ -105,6 +109,10 @@ public void backup(OutputStream os, AtomicLong entryCount) throws IOException {
     // Whether buffering is still active.
     AtomicBoolean bufferingActive = new AtomicBoolean(true);
 
+    // Start the timer for backup metrics and reset the entry counter.
+    Metrics.LAST_BACKUP_ENTRIES_COUNT.dec(Metrics.LAST_BACKUP_ENTRIES_COUNT.getCount());
```

Comment:
Do we do this because a process could restore from backup multiple times?

Reply:
I don't think we would run into a scenario where we restored from backup multiple times. The idea was to just keep this in line with the regular backup metric information. I can remove it if you feel that it is unnecessary.